app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -37,4 +45,59 @@ object Performance {
             activity.reportFullyDrawn()
         }
     }
+
+    /**
+     * Processes intent for Performance testing to remove protection pop up ( but keeps the TP
+     * on) and removes the onboarding screen.
+     */
+    fun processIntentIfPerformanceTest(intent: Intent, context: Context) {
+        if (isPerformanceTest(intent)) {
```

Comment:
nit: !isPerformanceTest

Reply:
I kept the intent with the !, so I didn't add it here, which in hindsight isn't quite obvious